client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -55,9 +58,27 @@ export default React.createClass( {
 		);
 	},
 
+	renderStoreStep() {
+		switch ( abtest( 'signupStoreBenchmarking' ) ) {
+			case 'bluehost':
+			case 'bluehostWithWoo':
+				return (
+					<BluehostStoreStep { ... this.props } onBackClick={ this.handleStoreBackClick }/>
```

Comment:
I think we typically leave a space before the closing of an inline tag, like:

`onBackClick={ this.handleStoreBackClick } />`

Correct me if I'm wrong @dmsnell @rodrigoi ;)